SWLOR.Game.Server/Feature/RecipeDefinition/CookingRecipeDefinition/CookingRecipes.cs

```diff
@@ -1303,6 +1303,18 @@ private void Tier4()
                 .Component("sanddemon_meat", 3)
                 .Component("r_flour", 2);
 
+            _builder.Create(RecipeType.DathomirPie, SkillType.Agriculture)
```

Comment:
Can you add this to the bible and send that over to me?

Reply:
https://docs.google.com/spreadsheets/d/1KBY3SuVAfFoh3Rt9aH9JPRY1ldzvDmhQPmzXHeHIhm8/edit#gid=1610037751
Dathomir Pie added
Dantooine Flapjack  items updated to reflect the recipe
Krafters Kebab items updated too